packages/web/components/elements/ExtensionsInstallHelp.tsx

```diff
@@ -106,8 +112,9 @@ export default function ExtensionsInstallHelp(): JSX.Element {
             height: '130px',
           }}
         >
-          <img srcSet="/static/images/extension-preview.png,
-             /static/images/extension-preview@2x.png 2x" />
+          <img
```

Comment:
Do we need this change? It will remove the high resolution image on higher density screens.

Reply:
Ah I see. Its a new image, we need to update the high resolution version too.